### DISCUSSION TITLE
Default sorting settings for solutions

### DISCUSSION THREAD

**Discussion by Captain-Quack** [codewars/codewars.com#2916] [Ideas]
(*this is a standalone idea from [this](https://github.com/codewars/codewars.com/discussions/2888) list*)
As Kacarott mentioned in the original post, it might be nice to be able to set a default sorting setting for solutions. Some users might want to see the most recent post, some might want to see what their friends did, and others might want to see a better way they could have solved the kata. 

Also, an option to see all language solutions side-by-side could be added? Quickly comparing different languages is one of my favorite features of CW, and it would be neat to see all of them at once.

**Comment by futter-krot** [comment]
Yeah, that would be really nice!
